lib/portlayer/storage/vsphere/volume.go

```diff
@@ -100,12 +106,13 @@ func (v *VolumeStore) VolumeCreate(op trace.Operation, ID string, store *url.URL
 	// Get the path to the disk in datastore uri format
 	volDiskDsURL := v.volDiskDsURL(ID)
 
+	config := disk.NewPersistentDisk(volDiskDsURL).WithCapacity(int64(capacityKB))
```

Comment:
Means we cannot use these calls to construct the base disk spec for scratch, meaning even more of a special case.